test/android/core_internal/javatests/src/org/xwalk/core/internal/xwview/test/XWalkViewInternalTestBase.java

```diff
@@ -241,7 +241,30 @@ protected void setUp() throws Exception {
             @Override
             public void run() {
                 mXWalkViewInternal = new XWalkViewInternal(activity, activity);
-                getActivity().addView(mXWalkViewInternal);
+                // Create new FrameLayout as a container to add XWalkViewInternal and
+                // ContentViewRenderView, it is just for test case
+                // "OnShowOnHideCustomViewTest.testOnShowCustomViewAndPlayWithHtmlControl".
+                // The play button should be rendered above the video right in
```

Comment:
`XWalkView` does not display in the middle of the screen without `FrameLayout`.
With following code to force `XWalkView` to display in the middle.
`                         new FrameLayout.LayoutParams(
                                 FrameLayout.LayoutParams.MATCH_PARENT,
                                 FrameLayout.LayoutParams.MATCH_PARENT,
                                 Gravity.CENTER_HORIZONTAL|Gravity.CENTER_VERTICAL)`.